GenericDiffractometer.py

```diff
@@ -289,9 +286,10 @@ def init(self):
             if self.transfer_mode is None or self.transfer_mode == "SAMPLE_CHANGER":
                 self.use_sc = True
 
-
         # centring motors
         self.centring_motors_list = ["phiz", "phiy","sampx","sampy"]
+        queue_model_objects.CentredPosition.\
```

Comment:
The list of centring motors should come from xml (line 238), because each site has a different motors used for centring. If directly method set_diffractometer_motor_names is called then this should be generic.
I will accept this PR and fix this